Multi-Chain/osmosis-cosmos-bridge/schema.graphql

```diff
@@ -0,0 +1,18 @@
+# To improve query performance, we strongly suggest adding indexes to any field that you plan to filter or sort by
+# Add the `@index` or `@index(unique: true)` annotation after any non-key field
+# https://academy.subquery.network/build/graphql.html#indexing-by-non-primary-key-field
+
+type CosmosToOsmosisBridge @entity {
```

Comment:
Can you Rename this simple to `BridgeEvent`
`CosmosToOsmosisBridge` kinda infers that this is one-directional between Cosmos -> Osmosis
Then in the future it's easy for us to add other networks to this project